apps/trading/lib/utils/get-erc20-abi.ts

```diff
@@ -0,0 +1,21 @@
+import { type Address, erc20Abi } from 'viem';
+import { ERC20_TEHTER_ABI } from '@vegaprotocol/smart-contracts';
+
+const TETHER_ADDRESS = '0xdAC17F958D2ee523a2206206994597C13D831ec7';
```

Comment:
chore (non-blocking): think this would make more sense as an array present in an env var rather than hard coded into the app.

Reply:
Prefer keeping things out of the env file and in code where possible. Ideally each env file could have a single vega chain id and ethereum chain id and everything can be keyed from that